.github/workflows/ci.yaml

```
@@ -0,0 +1,67 @@
+name: DEV-CI
+
+on: 
+  push:
+    branches:
+      - dev
+
+jobs:
+  ci:
+    name: DEV-CI
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+      id-token: write
+      packages: write
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: Run test
+      run: make test
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v3
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    - name: Build_Push
+      env:
+        APP: "ibot"
+        REGESTRY: "ghcr.io/k3ilona"
+      run: make image push
+    - name: Clean
+      run: make clean
+      
+  helm:
+    name: DEV-HELM
+    needs: ci
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: Define tag
+      run: echo "TAG=$(git describe --tags --abbrev=0)" >> $GITHUB_ENV
+
+    - name: Define version
+      run: echo "VERSION=$TAG-dev-$(git rev-parse --short HEAD)" >> $GITHUB_ENV
```

Щоб не хардкодити бранч можна внести наступні зміни @dm-ol 

```suggestion
      run: echo "VERSION=$TAG-${{ github.ref_name }}-$(git rev-parse --short HEAD)" >> $GITHUB_ENV
```